lib/landing_session.js

```diff
@@ -157,16 +176,36 @@ class LandingSession extends Session {
 
   async apply() {
     const { cli } = this;
+
+    // Bail if another landing session is currently in progress.
     if (!this.isApplying()) {
-      cli.warn('This session can not proceed to apply patches, ' +
-        'run `git node land --abort`');
+      cli.warn('Landing session already in progress - ' +
+        'to start a new one run `git node land --abort`');
       return;
     }
     await this.tryResetBranch();
 
     const patch = await this.downloadAndPatch();
+
+    const cleanLint = await this.validateLint(patch);
+    if (!cleanLint) {
+      const correctedLint = await cli.prompt('Corrected lint errors?');
```

Comment:
Sorry if I missed something but, is a user supposed to fix the issues in another terminal and then come back to this?

Should we perhaps try to run `make lint-js-fix` for the user here?

Reply:
@lundibundi yes - the intent is that they fix it elsewhere and continue. We could potentially try to run `make lint-js-fix` though - i'm not opposed to doing that as well!

Reply:
added